@navikt/ds-css/grid/cell.css

```diff
@@ -0,0 +1,209 @@
+.navds-grid-cell {
+  border-radius: 0.25rem;
+}
+
+.navds-grid-cell--white {
```

Comment:
Selv tenker jeg at griden ikke burde ha noe med farge eller border-radius å gjøre, men at det er opp til innholdet man fyller griden med. Men her kan jeg sikkert overbevises.

Reply:
Samme gjelder vel egentlig for padding.

Reply:
Det er et godt poeng det, kan endre det og legge stylingen på i eksempelet :)